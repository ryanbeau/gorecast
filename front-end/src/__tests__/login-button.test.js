import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import LoginButton from "../components/login-button.js";

it("renders correctly", () => {
  const wrapper = shallow(<LoginButton />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
