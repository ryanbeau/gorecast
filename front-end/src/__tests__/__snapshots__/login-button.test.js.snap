// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="btn btn-primary btn-block"
  onClick={[Function]}
>
  Log In
</button>
`;
